documentation/kit-artifacts.md

```diff
@@ -0,0 +1,210 @@
+Deliverables:
+
+- Description
+- Tutorial video or screenshot documentation
+
+### Whitepaper
+
+Optional
+A whitepaper outlines the overall objectives of a KIT regarding a specific business problem. It provides readers with background information on the topic, identifies key challenges and solutions to those challenges, evaluates alternative approaches to solving the business problem at hand, and offers recommendations for action.
+Deliverable:
+
+- Description
+- Link to the whitepaper
+
+### Semantic Models
+
+Mandatory for every KIT that is related to a use case + for the 'incubating' stage
+The semantic data models are a method of structuring data that includes semantic information that adds a basic meaning to the data and the relationships that lie between them to enable interoperability.
+Deliverables:
+
+- Description
+- Link to the semantic data models (GitHub)
+`Example from Traceability:`
+[Traceability Semantic Models](https://eclipse-tractusx.github.io/docs-kits/kits/Traceability%20Kit/Adoption%20View%20Traceability%20Kit#semantic-models)
+
+### Logic / Schema
+
+Mandatory for every KIT that is related to a use case + for the 'incubating' stage
+Logic / schemas provides a definition of the minimum viable business process that must be implemented to enable interoperability.
+Deliverables: Description
+
+### Business Process
+
+Mandatory for every KIT that is related to a use case + for the 'incubating' stage
+Data sovereignty is one of the main principles of Catena-X and a major topic for the participants in the data space. This part should help to understand how the business process works and provides a blue print on how access & usage policies **could** look like.
+
+- Business Architecture
+- Access & Usage Policies
+
+### Standards
+
+Mandatory for every KIT that wants to go to the 'incubating' stage
+Links to the relevant standards (Association Library). Add the relevant number of the standard, the name and version of the standard.
+`Example:`
+CX - 001 Standardname (Version 2.0)
+
+### Notice
+
+Mandatory for every KIT and all graduation stages
+KIT documentation works under the CC-BY-4.0 license. Therefore you need to add the "notice" part to make transparent which companies worked on this KIT.
+`Example:`
+This work is licensed under the [CC-BY-4.0](https://creativecommons.org/licenses/by/4.0/legalcode).
+
+- SPDX-License-Identifier: CC-BY-4.0
+- SPDX-FileCopyrightText: 2023,2023 ZF Friedrichshafen AG
+- SPDX-FileCopyrightText: 2023,2023 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+- SPDX-FileCopyrightText: 2023,2023 SAP SE
+- SPDX-FileCopyrightText: 2023,2023 Volkswagen AG
+- SPDX-FileCopyrightText: 2023,2023 Robert Bosch GmbH
+- SPDX-FileCopyrightText: 2023,2023 Mercedes Benz Group
+- SPDX-FileCopyrightText: 2023,2023 BASF SE
+- SPDX-FileCopyrightText: 2023,2023 Schaeffler AG
+- SPDX-FileCopyrightText: 2023,2023 Contributors to the Eclipse Foundation
```

Comment:
`2023,2023` is the wrong format and not in line with the Eclipse Foundation Project Handbook: https://www.eclipse.org/projects/handbook/#ip-copyright-headers

Reply:
@Unsharm please correct the example. If you have questions, feel free to contact @SujitMBRDI or me directly.